test/test.cc

```diff
@@ -8010,6 +8010,68 @@ TEST(MultipartFormDataTest, ContentLength) {
   ASSERT_TRUE(send_request(1, req, &response));
   ASSERT_EQ("200", response.substr(9, 3));
 }
+
+TEST(MultipartFormDataTest, AccessPartHeaders) {
+  auto handled = false;
+
+  Server svr;
+  svr.Post("/test", [&](const Request &req, Response &) {
+    ASSERT_EQ(2u, req.files.size());
+
+    auto it = req.files.begin();
+    ASSERT_EQ("text1", it->second.name);
+    ASSERT_EQ("text1", it->second.content);
+    ASSERT_EQ(1, it->second.headers.count("Content-Length"));
+    auto content_lenght = it->second.headers.find("CONTENT-length");
+    ASSERT_EQ("5", content_lenght->second);
+    ASSERT_EQ(3, it->second.headers.size());
+
+    ++it;
+    ASSERT_EQ("text2", it->second.name);
+    ASSERT_EQ("text2", it->second.content);
+    auto &headers = it->second.headers;
+    ASSERT_EQ(3, headers.size());
+    auto customHeader = headers.find("x-whatever");
```

Comment:
Could you rename `customHeader` to be `custom_header`, since we use the 'snake_case' instead of 'camelCase' for local variables.